nuget/Plugin.nuspec

```diff
@@ -59,7 +59,8 @@
     <file src="src\ExternalMaps.Plugin.UWP\bin\Release\Plugin.ExternalMaps.*" target="lib\UAP10" />
     <!--Xamarin.Mac-->
     <file src="src\ExternalMaps.Plugin.Mac\bin\Release\Plugin.ExternalMaps.*" target="lib\Xamarin.Mac20" />
-
+    <!--Tizen-->
```

Comment:
We need to add a new dependency group above with all of the required NuGets for the Tizen version.